### PR TITLE
[doc] Fix docs deadlink in readme

### PR DESCRIPTION
**Pull request by songjianet**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by labbomb — DISMISSED** [review]
LGTM

**Review by devosend — DISMISSED** [review]
LGTM

**Review by zhongjiajie — COMMENTED** [review]
could we use related path to use the link?

File: README.md [hunk 1]
```diff
@@ -54,11 +54,11 @@ Overload processing: By using the task queue mechanism, the number of schedulabl
 
 ## User Interface Screenshots
 
-![dag](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/dag.png)
-![data-source](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/data-source.png)
-![home](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/home.png)
-![master](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/master.png)
-![workflow-tree](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/workflow-tree.png)
+![dag](https://github.com/apache/dolphinscheduler/blob/dev/images/en_US/dag.png)
```
Comment (zhongjiajie) [inline README.md]:
```suggestion
![dag](./images/en_US/dag.png)
```

Comment (zhongjiajie) [reply README.md]: such as using this

File: README.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ Overload processing: By using the task queue mechanism, the number of schedulabl
...
-![dag](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/dag.png)
-![data-source](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/data-source.png)
-![home](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/home.png)
-![master](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/master.png)
-![workflow-tree](https://github.com/apache/dolphinscheduler/tree/dev/images/en_US/workflow-tree.png)
+![dag](./images/en_US/dag.png)
+![data-source](https://github.com/apache/dolphinscheduler/blob/dev/images/en_US/data-source.png)
+![home](https://github.com/apache/dolphinscheduler/blob/dev/images/en_US/home.png)
+![master](https://github.com/apache/dolphinscheduler/blob/dev/images/en_US/master.png)
+![workflow-tree](https://github.com/apache/dolphinscheduler/blob/dev/images/en_US/workflow-tree.png)
```
Comment (zhongjiajie) [inline README.md]:
```suggestion
![data-source](./images/en_US/data-source.png)
![home](./images/en_US/home.png)
![master](./images/en_US/master.png)
![workflow-tree](./images/en_US/workflow-tree.png)
```

File: README_zh_CN.md [hunk 3]
```diff
@@ -36,11 +36,11 @@ Dolphin Scheduler Official Website
...
-![dag](https://github.com/apache/dolphinscheduler/tree/dev/images/zh_CN/dag.png)
-![data-source](https://github.com/apache/dolphinscheduler/tree/dev/images/zh_CN/data-source.png)
-![home](https://github.com/apache/dolphinscheduler/tree/dev/images/zh_CN/home.png)
-![master](https://github.com/apache/dolphinscheduler/tree/dev/images/zh_CN/master.png)
-![workflow-tree](https://github.com/apache/dolphinscheduler/tree/dev/images/zh_CN/workflow-tree.png)
+![dag](https://github.com/apache/dolphinscheduler/blob/dev/images/zh_CN/dag.png)
+![data-source](https://github.com/apache/dolphinscheduler/blob/dev/images/zh_CN/data-source.png)
+![home](https://github.com/apache/dolphinscheduler/blob/dev/images/zh_CN/home.png)
+![master](https://github.com/apache/dolphinscheduler/blob/dev/images/zh_CN/master.png)
+![workflow-tree](https://github.com/apache/dolphinscheduler/blob/dev/images/zh_CN/workflow-tree.png)
```
Comment (zhongjiajie) [inline README_zh_CN.md]:
```suggestion
![dag](./images/zh_CN/dag.png)
![data-source](./images/zh_CN/data-source.png)
![home](./images/zh_CN/home.png)
![master](./images/zh_CN/master.png)
![workflow-tree](./images/zh_CN/workflow-tree.png)
```

**Review by zhongjiajie — APPROVED** [review]

**Pull request merged**